src/Nethermind/Nethermind.Core/BaseFeeCalculator.cs

```diff
@@ -11,32 +11,37 @@ public static class BaseFeeCalculator
     {
         public static UInt256 Calculate(BlockHeader parent, IEip1559Spec specFor1559)
         {
-            UInt256 expectedBaseFee = parent.BaseFeePerGas;
+            return Calculate(parent.BaseFeePerGas, parent.GasUsed, parent.GasLimit, parent.Number, specFor1559);
```

Comment:
why was this extra call added ?

Reply:
thought it would be convenient at some point, now reverted